Apps/phironnet/ironnet.json

```diff
@@ -1290,6 +1290,14 @@
                         "https://1.2.3.4/event?uuid=9f61e9e9af904540ac8bd730f5ed48d2"
                     ]
                 },
+                {
+                    "data_path": "action_result.data.*.events.*.src_network_id",
+                    "data_type": "string"
+                },
```

Comment:
Do we need to have "example_values" similar to the other fields?

Reply:
i think you're right i should add these--was looking at the examples before mike's new PR Comments PR which didn't always include examples --my bad. will add.